src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx

```diff
@@ -71,40 +76,61 @@ class BreadcrumbsContainer extends React.Component<Props, State> {
     }
 
     const breadcrumbTypes: BreadcrumbFilterGroups = [];
-
-    // TODO(Priscila): implement levels
-    //const breadcrumbLevels: BreadcrumbFilterGroups = [];
+    const breadcrumbLevels: BreadcrumbFilterGroups = [];
 
     const convertedBreadcrumbs = breadcrumbs.map((breadcrumb, index) => {
       const convertedBreadcrumb = convertBreadcrumbType(breadcrumb);
-      const breadcrumbDetails = getBreadcrumbDetails(convertedBreadcrumb.type);
+      const breadcrumbTypeDetails = getBreadcrumbTypeDetails(convertedBreadcrumb.type);
 
       if (!breadcrumbTypes.find(b => b.type === convertedBreadcrumb.type)) {
         !breadcrumbTypes.push({
           groupType: FilterGroupType.TYPE,
           type: convertedBreadcrumb.type,
-          ...breadcrumbDetails,
+          description: breadcrumbTypeDetails.description,
+          symbol: (
+            <BreadcrumbIcon
+              icon={breadcrumbTypeDetails.icon}
+              color={breadcrumbTypeDetails.color}
+              borderColor={breadcrumbTypeDetails.borderColor}
```

Comment:
Should these be destructured? It could also help a little with @EvanPurkhiser 's concern about verbosity.